### PR TITLE
P0/archive notes

### PR DESCRIPTION
**Pull request by purvasharmadev**
## Added Archive feature

- user can archive a note
- user can restore the archive note

### CODE REVIEW

**Review by adityapandadev — APPROVED** [review]

**Review by saurabhan — APPROVED** [review]
Well written Code, everything work as expected,
Do remove Console logs from Prod build.

File: src/Context/archive-context.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+  async function getArchive() {
+    try {
+      const res = await axios.get("/api/archives", {
+        headers: {
+          "content-type": "text/json",
+          authorization: localStorage.getItem("userToken"),
+        },
+      });
+      console.log("archive ", res);
```
Comment (zoyatanweer) [inline src/Context/archive-context.js:20]: remove console

File: src/Context/archive-context.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+          },
+        },
+        {
+          headers: {
+            "content-type": "text/json",
+            authorization: localStorage.getItem("userToken"),
+          },
+        }
+      );
+      console.log("archives post ", res);
```
Comment (zoyatanweer) [inline src/Context/archive-context.js:45]: remove

File: src/Context/archive-context.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        `/api/archives/restore/${id}`,
+        {},
+        {
+          headers: {
+            "content-type": "text/json",
+            authorization: localStorage.getItem("userToken"),
+          },
+        }
+      );
+      console.log("archive restore ", res);
```
Comment (zoyatanweer) [inline src/Context/archive-context.js:81]: remove console

File: src/Context/notes-context.js [hunk 4]
```diff
@@ -67,67 +73,71 @@ function NotesProvider({ children }) {
...
-      )
-      console.log("edit ",res.data.notes)
-      dispatch({type:"notes_LIST",payload:res.data.notes})
-      navigateTo("/notes")
-      dispatch({type:"CLEAR_notes"})
-      
-    }catch(error){
-        console.error(error.resposne.data.errors[0])        
+      );
+      console.log("edit ", res.data.notes);
```
Comment (zoyatanweer) [inline src/Context/notes-context.js:97]: remove console

File: src/Context/notes-context.js [hunk 5]
```diff
@@ -67,67 +73,71 @@ function NotesProvider({ children }) {
...
 
   // Saving notes in localStorage
   useEffect(() => {
     // getNotes();
     localStorage.setItem("notes", JSON.stringify(state.notesList));
   }, [state.notesList]);
 
-//   postNotes();
+  //   postNotes();
   console.log("state ", state);
```
Comment (zoyatanweer) [inline src/Context/notes-context.js:127]: remove console

**Review by zoyatanweer — APPROVED** [review]
- code quality is nice
- suggestion: add comments where the component starts
- good to merge

**Pull request merged**